src/app/pages/TopicPage/Curation/index.jsx

```diff
@@ -40,11 +41,15 @@ const Curation = ({
   const createID = titleText => {
     return titleText.replaceAll(' ', '-');
   };
-  return curationLength > 1 && (title || position === 0) ? (
-    <section aria-labelledby={createID(title || topStoriesTitle)} role="region">
-      <Subheading a11yID={createID(title || topStoriesTitle)} href={link}>
-        {title}
-      </Subheading>
+  const isFirstCuration = position === 0;
+  const SubheadingComponent = isFirstCuration ? VisuallyHiddenText : Subheading;
+  const id = createID(title || topStoriesTitle);
+
+  return curationLength > 1 && (title || isFirstCuration) ? (
+    <section aria-labelledby={id} role="region">
+      <SubheadingComponent as="h2" a11yID={id} id={id} href={link}>
```

Comment:
I don't think this would ever be a link

Reply:
If it is the first curation, we render the VIsuallyHiddenText component as a h2. If it is second+ curation, we render the Subheading component which can contain a link. It is the Subheading component that accepts href and a11yID as props

Reply:
Ok, currently if I have looked at the correct storybook link the h2 is output with an empy href?

Reply:
Prop name updated